discovery-provider/alembic/versions/ed974e76d415_fix_indexing_track_slug.py

```diff
@@ -0,0 +1,405 @@
+"""fix-indexing-track-slug
+
+Revision ID: ed974e76d415
+Revises: cb9aa46f1e46
+Create Date: 2021-07-30 15:24:52.899198
+
+"""
+from alembic import op
+from src.models import Block, Track, TrackRoute, SkippedTransaction
```

Comment:
I think we should bring a copy of these models into this file as migrations can't reliably depend on models not changing

Reply:
Good point, can do